tests/wpt/mozilla/tests/css/text_overflow_string.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel=mismatch href=text_overflow_ref.html>
+<style>
+p {
+    width: 128px;
+    background: gold;
+}
+p.hidden {
+    overflow: hidden;
+}
+.overflow-string {
```

Comment:
Same here, apply `text-overflow` to both paragraphs, then `overflow: hidden` to just one.